features/step_definitions/restaurant_list_steps.rb

```diff
@@ -3,3 +3,19 @@
     FactoryBot.create(:restaurant, restaurant)
   end
 end
+
+Then("show me the page") do
```

Comment:
This step definition should be removed

Reply:
This has been removed in feature file but would prefer to keep it for future step defs. Thanks!